RInAppMessaging/Classes/Models/UserInfoProvider.swift

```diff
@@ -0,0 +1,35 @@
+/**
+ * Interface which client app should implement in order for InAppMessaging SDK to get information
+ * when needed.
+ */
+@objc public protocol UserInfoProvider {
+
+    /**
+     * Only return RAE token if user is logged in. Else return null.
+     *
+     * @return String of RAE token.
+     */
+    var provideRaeToken: String? { get }
```

Comment:
@donnie-jp on iOS we used to use `accessToken` name for that property. Should we keep using that name to avoid confusing customers or use `raeToken` to keep the name consistent across both platforms?

Reply:
I don't really like having references to RAE in our open source SDK (not for security reasons but cos it's too "internal") but naming this is tricky.

Maybe this could be `provideOAuthToken` and then in the future we can add `provide{auth scheme name}Token` e.g., `provideOpenIDToken` - what do you think?

Reply:
@donnie-jp that's a good idea. I think we shouldn't stick to RAE names too much because soon they'll become obsolete